docs/help/_posts/2010-05-10-configure.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: Safe configuration and overlapping calls
+layout: post
+---
+
+Sometimes we want to configure a call that overlaps a more general call we have previously setup to run a [callback](/help/callbacks) or throw an [exception](/help/throwing-exceptions/). Ideally we would modify the setups so they don't overlap, but we can also prevent these callbacks from running while we setup the next call by calling `Configure()` before performing additional `Returns()`. `Configure()` tells NSubstitute we are configuring a call so that it will not run any callbacks from previous configurations.
```

Comment:
> by calling `Configure()` before performing additional `Returns()`

From description looks like we are doing smth like `subs.Method().Configure().Returns()`. Better to rephrase to e.g. "before invoking the method we want to re-configure" or something similar.